shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt

```diff
@@ -47,23 +48,23 @@ open class Command(initialCommand: List<String>) {
         return this
     }
 
-    open fun runProcess(): Int {
+    open fun runProcess() {
         val builder = ProcessBuilder(command)
 
         builder.redirectOutput(Redirect.INHERIT)
         builder.redirectInput(Redirect.INHERIT)
-        builder.redirectError(Redirect.INHERIT)
 
         val process = builder.start()
+
+        val reader = BufferedReader(InputStreamReader(process.errorStream))
+
         val exitCode = process.waitFor()
-        return exitCode
+        handleExitCode(exitCode, reader.readLines())
```

Comment:
It doesn't return anything, while it's required in Linker.kt

Reply:
See https://buildserver.labs.intellij.net/viewLog.html?buildId=66946137&problemId=1583664